prql-compiler/prqlc/src/cli.rs

```diff
@@ -477,4 +502,19 @@ group a_column (take 10 | sort b_column | derive [the_number = rank, last = lag
           - !Single y
         "###);
     }
+    #[test]
+    fn get_targets() {
+        let n_targets = Target::names().len();
+
+        assert_cmd::Command::cargo_bin("prqlc")
+            .unwrap()
+            .args(["get-targets"])
+            .assert()
+            .success()
+            .stdout(
+                predicates::str::is_match(r"sql\.[a-z]+\n")
+                    .unwrap()
+                    .count(n_targets),
+            );
+    }
```

Comment:
This is failing because it requires a --release binary of prqlc ... which is called from a test in a --test prqlc binary.

We could probably make this work, but I'm not sure it's worth the headache.